one-ui/src/main/java/com/marklogic/hub/oneui/models/HubConfigSession.java

```diff
@@ -990,7 +1007,18 @@ private DatabaseClient getClientByDatabaseKindAndName(DatabaseKind kind, String
         return client;
     }
 
-    public DataHub getDataHub() {
-        return this.dataHub;
+    /**
```

Comment:
Is this code required ? Just configuring session timeout property not sufficient ? I would assume as HubConfigSession is a session scoped bean , just invalidating session would eventually destroy these objects ?

Reply:
I think code like this is a great argument for a HubClient class that only contains DatabaseClient objects and none of the original state. "reset" methods to me are an anti-pattern, particularly because it means that any time more fields are added to an object, someone has to remember to consider whether the new fields need to be reset or not.

Reply:
Actually, I do not think we need to add "reset" here after I checked in...